conda/template/meta.yaml

```diff
@@ -23,9 +23,11 @@ build:
 
 requirements:
   build:
+    - openmpi
```

Comment:
Good call. Originally we were going to need to switch to OpenMPI because we didn't know why latest versions of MPICH was hanging.

That was until @lindsayad discovered the following solves that issue:
```bash
 FI_PROVIDER=tcp
```
*which is now being set in our `moose-mpi` package.*

I agree, we should switch this back to mpich!